--- git status ---
On branch main
Your branch is ahead of 'origin/main' by 1 commit.
  (use "git push" to publish your local commits)

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   source/limx_rl_forge/limx_rl_forge/tasks/locomotion/robots/oli/31dof/velocity_env_cfg.py

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	humanoid-description
	logs/
	outputs/
	scripts/rsl_rl/__pycache__/
	source/limx_rl_forge/limx_rl_forge/__pycache__/
	source/limx_rl_forge/limx_rl_forge/assets/__pycache__/
	source/limx_rl_forge/limx_rl_forge/assets/config/__pycache__/
	source/limx_rl_forge/limx_rl_forge/tasks/__pycache__/
	source/limx_rl_forge/limx_rl_forge/tasks/locomotion/__pycache__/
	source/limx_rl_forge/limx_rl_forge/tasks/locomotion/agents/__pycache__/
	source/limx_rl_forge/limx_rl_forge/tasks/locomotion/mdp/__pycache__/
	source/limx_rl_forge/limx_rl_forge/tasks/locomotion/robots/__pycache__/
	source/limx_rl_forge/limx_rl_forge/tasks/locomotion/robots/oli/31dof/__pycache__/
	source/limx_rl_forge/limx_rl_forge/tasks/locomotion/robots/oli/__pycache__/
	source/limx_rl_forge/limx_rl_forge/tasks/locomotion/terrain/__pycache__/
	wandb/

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/source/limx_rl_forge/limx_rl_forge/tasks/locomotion/robots/oli/31dof/velocity_env_cfg.py b/source/limx_rl_forge/limx_rl_forge/tasks/locomotion/robots/oli/31dof/velocity_env_cfg.py
index 324892b..d88f340 100644
--- a/source/limx_rl_forge/limx_rl_forge/tasks/locomotion/robots/oli/31dof/velocity_env_cfg.py
+++ b/source/limx_rl_forge/limx_rl_forge/tasks/locomotion/robots/oli/31dof/velocity_env_cfg.py
@@ -349,7 +349,7 @@ class VelocityRewardsCfg:
         func=mdp.feet_gait,
         weight=0.5,
         params={
-            "period": 1.2,
+            "period": 0.667,
             "offset": [0.0, 0.5],
             "threshold": 0.55,
             "command_name": "base_velocity",